src/Build/BuildCheck/API/BuildAnalyzerConfiguration.cs

```diff
@@ -84,14 +84,13 @@ internal static BuildAnalyzerConfiguration Create(Dictionary<string, string>? co
 
         switch (stringValue)
         {
-            case "project":
-                return BuildCheck.EvaluationAnalysisScope.ProjectOnly;
-            case "current_imports":
-                return BuildCheck.EvaluationAnalysisScope.ProjectWithImportsFromCurrentWorkTree;
-            case "without_sdks":
-                return BuildCheck.EvaluationAnalysisScope.ProjectWithImportsWithoutSdks;
+            case "projectfile":
+            case "project_file":
```

Comment:
Why both options are supported and why don't we pick the uniform style for the parameter?

Reply:
This is a good point.

I thought both ("Project file" and "Projectfile") are correct in English. It turns out that likely just the first one. I'm adjusting the documentation - we should support only `"project_file"`.
I'm keeping the `projectfile` in code to grace handle misstypes by ppl misslead similarly like me.